__tests__/index.test.ts

```diff
@@ -288,16 +284,18 @@ describe('Auth0', () => {
       };
 
       xit('calls oauth/token without redirect uri if not set in transaction', async () => {
-        const { auth0, utils, transactionManager } = await localSetup();
-        const txn = transactionManager.get.mockReturnValue({
+        const { auth0, transactionManager, api } = await localSetup();
+
+        transactionManager.get.mockReturnValue({
           code_verifier: TEST_RANDOM_STRING,
           nonce: TEST_RANDOM_STRING,
           audience: 'default',
           scope: TEST_SCOPES,
           appState: TEST_APP_STATE
         });
+
         await auth0.handleRedirectCallback();
-        const arg = utils.oauthToken.mock.calls[0][0];
+        const arg = api.oauthToken.mock.calls[0][0];
```

Comment:
This test is excluded, is that expected? Should we keep it ? If not, we do not need to return api from the localSetup.

Reply:
Not sure why it's being skipped, will investigate.

Reply:
I'm guessing also that these tests should be part of `handleRedirectCallback.test.ts`, doesn't look like they were moved over.